src/main/java/uk/ac/ebi/eva/contigalias/controller/contigalias/ContigAliasController.java

```diff
@@ -211,7 +215,10 @@ public static void linkPagedModelGetChromosomesByAssemblyAccession(
     @GetMapping(value = "assemblies/{accession}/chromosomes", produces = "application/json")
     public ResponseEntity<PagedModel<EntityModel<ChromosomeEntity>>> getChromosomesByAssemblyAccession(
             @PathVariable String accession,
-            @RequestParam(required = false, name = "authority") String chrAuthority,
+            @RequestParam(required = false, name = "authority") @ApiParam("Specify if the provided accession is a " +
+                    "GenBank or a RefSeq accession. The acceptable param values are " + AUTHORITY_GENBANK + " " +
+                    "and " + AUTHORITY_REFSEQ + " respectively. If this parameter is omitted then the accession is " +
+                    "assumed to have an authority of " + AUTHORITY_GENBANK + " by default.") String chrAuthority,
```

Comment:
> If this parameter is omitted then the accession is assumed to have an authority of " + AUTHORITY_GENBANK

is this right? the code seems to search in both if the authority is not present. However, in line 240 there's no call to `linkPagedModelGetAssemblyByChromosomeAuthority()` as in the other cases.

Reply:
Somehow I was confused about the accession and authority being parameters of the assembly or chromosome and was linking the wrong methods to the paged models. I have fixed the linking now. I've also fixed the @ApiParam to tell the user that the accession will be searched as being either a GenBank or a RefSeq accession.